packages/next/package.json

```diff
@@ -152,6 +152,7 @@
     "@types/lodash.curry": "4.1.6",
     "@types/lru-cache": "5.1.0",
     "@types/micromatch": "4.0.2",
+    "@types/node": "13.11.0",
```

Comment:
The existing version did not include `os.version()`

Reply:
What was the existing version? It seems like its resolved from somewhere else?

Reply:
`13.1.4`. Lowest supported version is `13.11.0`

https://nodejs.org/api/os.html#osversion

Reply:
It says `v12.17.0` in the doc you linked to.

Regardless, I think the next/package.json is the wrong place to update this dev dep since its being added, not updated.

Can you try updating this one instead to 12.22.0 so it matches `engines`? https://github.com/vercel/next.js/blob/5378db8f807dbb9ff0993662f0a39d0f6cba2452/packages/create-next-app/package.json#L31

Reply:
Yeah, I just did not want to downgrade at least. Will upgrade the existing one instead. :+1:

Reply:
So there is no `12.22.0` version.

Snippet of available ones:

```
13.1.6 
❯ 13.1.5 
  13.1.4 
  13.1.3 
  13.1.2 
  13.1.1 
  13.1.0 
  12.20.40 
  12.20.39 
  12.20.38 
```

Reply:
Lets use 14.x then

Reply:
Specifying `14.0.0` does not resolve the `os.version()` issue in `packages/next` and creates more TS conflicts elsewhere.

Reply:
Not 14.0.0 but whatever the latest 14 is. 

Look like 14.18.4